definitions/2.6.0/server.json

```diff
@@ -13,38 +13,48 @@
   },
   "properties": {
     "url": {
-      "type": "string"
+      "type": "string",
+      "description": "A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the AsyncAPI document is being served."
```

Comment:
I don't agree that we should call this a URL, but I'm being a pedant... plus that's already in https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#server-object so this isn't the PR to fix that in :-)